managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/MetaStoreImpl.java

```diff
@@ -455,7 +456,7 @@ public ManagedLedgerInfo parseManagedLedgerInfo(byte[] data) throws InvalidProto
                         + "fall back to parse managedLedgerInfo directly.", e);
                 return ManagedLedgerInfo.parseFrom(data);
             } finally {
-                byteBuf.release();
+                ReferenceCountUtil.safeRelease(byteBuf);
```

Comment:
This PIP has not passed yet. The disscuss thread: https://lists.apache.org/thread/c024dyn3gtfymdssqrqmg3s9gtn4ths9